tests/rptest/services/kerberos.py

```diff
@@ -457,6 +457,14 @@ def clean_node(self, node, **kwargs):
             node.account.ssh(f"rm -fr {self.krb5_conf_path}", allow_fail=True)
 
     def start_node(self, node, **kwargs):
+        if self.krb5_conf_path != KRB5_CONF_PATH:
+            self.logger.debug(
+                f"Rendering incorrect KRB5 config for {node.name} using KDC node {self.kdc.nodes[0].name}"
+            )
+            krb5_config = render_krb5_config(kdc_node=self.kdc.nodes[0],
+                                             realm="INCORRECT.EXAMPLE")
+            node.account.create_file(KRB5_CONF_PATH, krb5_config)
+
```

Comment:
I think it's a  little bit of a misdirection that just because you're not using `/etc/krb5.conf` as your config file, that implies that you're not using a valid one.  Maybe that should be a separate flag instead of `self.krb5_conf_path != KRB5_CONF_PATH`

Reply:
I moved this to the test.